### PR TITLE
XIVY-13820 introduce form-editor screenshots

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

File: build/screenshots/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            maven cmd: '-ntp -f playwright/tests/screenshots clean deploy -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
+          }
+          archiveArtifacts 'playwright/tests/screenshots/target/form-editor-screenshots-*.zip'
+        }
+      }
+    }
+  }
+  post {
+    always {
+      discoverGitReferenceBuild()
```
Comment (ivy-rew) [inline build/screenshots/Jenkinsfile:47]: copied: not sure what it is for :thinking:

Comment (ivy-lli) [reply build/screenshots/Jenkinsfile:47]: Not sure, for detect if git changes happened?

File: build/screenshots/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      }
+    }
+    stage('Deploy') {
+      when {
+        expression { isReleaseOrMasterBranch() && currentBuild.currentResult == 'SUCCESS' }
+      }
+      steps {
+        script {
+          docker.image('maven:3.8.6-eclipse-temurin-17').inside {
+            maven cmd: '-ntp -f playwright/tests/screenshots clean deploy -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
```
Comment (ivy-lli) [inline build/screenshots/Jenkinsfile:38]: clean... doesn't this delete the created screenshots?

File: playwright/tests/standalone/playwright.config.ts [hunk 3]
```diff
@@ -7,5 +7,14 @@ export default defineConfig(defaultConfig, {
...
+  },
+  projects: [
+    {
+      name: 'chromium',
+      use: {
+        viewport: { width: 700},
+        contextOptions: { reducedMotion: 'reduce' }
+      }
+    }
+  ]
```
Comment (ivy-lli) [inline playwright/tests/standalone/playwright.config.ts:19]: didn't you want to add this to the screenshot playwright config?

File: build/screenshots/Jenkinsfile (hunk 2 above)
Comment (ivy-rew) [reply build/screenshots/Jenkinsfile:38]: no; the screenshots are located on level higher ... so the clean doesn't really do anything ... but is normally good practice to use

**Pull request merged**